readme.txt

```diff
@@ -116,6 +116,57 @@ You'll find the [FAQ on Yoast.com](https://yoast.com/wordpress/plugins/seo/faq/)
 
 == Changelog ==
 
+= 3.4.0 =
+
+Release Date: July 19th, 2016
+
+* Enhancements:
+	* Adds the setting and user option to disable the SEO analysis.
+	* Adds the readability score to the post and term overview.
+	* Disables the analysis marker buttons when switching from visual to text view in the editor.
+	* Accessibility enhancements:
+		* Improves the headings in the dashboard widget.
+		* Improves the headings hierarchy on the following pages:
+			* titles and metas
+			* user profile
+			* advanced settings
+			* social settings
+			* XML sitemap
+			* general settings
+		* Improves the headings hierarchy for dashboard alerts.
+		* Improves the debug information headings.
+		* Adds a legend on the titles and metas page.
+		* Improves the intro text of the social tabs.
+		* Always use the buttons css class for buttons.
+		* Adds a background to the YoastSEO issue counter to improve readability.
+		* Improves the focus style for the dismiss and restore buttons.
+		* Improves the redirect attachment warnings style by using the native WordPress notices style.
+		* Improves the links on the credit screen.
+		* Improves the release video iframe by adding a title.
+		* Improves the knowledge base results by making them focusable and operable with a keyboard.
+		* Improves the admin bar menu items by making them focusable and operable with a keyboard.
+		* Adds labels to the buttons of the knowledge base search for use with a screen reader.
+		* Adds label to the search field in the knowledge base for screen readers.
+		* Makes the knowledge base search button translatable.
+		* Improves the semantics of the knowledge base results.
+	* Improves the alignment of the icons in the metabox tab sections.
+	* Don't do a strict check on blog_public
```

Comment:
You left this one, but I think you meant to delete it in favor of the following line.